packages/wallet-adapter-react/src/WalletProvider.tsx

```diff
@@ -54,6 +55,7 @@ const initialState: {
  * @param plugins Non AIP-62 supported wallet plugins array
  * @param optInWallets AIP-62 supported wallet names array to only include in the adapter wallets
  * @param autoConnect A boolean flag to indicate if the adapter should auto connect to a wallet
+ * @param aptosConnectConfig A JSON format configurations to connect with AptosConnect
```

Comment:
```suggestion
 * @param aptosConnectConfig configuration passed to the auto-injected AptosConnect plugin
```

also I wouldn't mention JSON if not in the context of serialization or transport